build/get_buildvers.py

```diff
@@ -34,7 +34,7 @@ def _get_buildvers(buildvers, pom_file, logger=None):
         else:
             no_snapshots.append(release)
     excluded_shims = pom.find(".//pom:dyn.shim.excluded.releases", ns)
-    if excluded_shims:
+    if excluded_shims is not None:
```

Comment:
`if excluded_shims:`

 was ineffective in checking whether the pom <Element> is None.

Reply:
TIL https://stackoverflow.com/a/3965165